argilla-server/src/argilla_server/jobs/queues.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import redis
+
+from rq import Queue
+
+from argilla_server.settings import settings
+
+
+redis_connection = redis.from_url(settings.redis_url)
+
+default_queue = Queue(connection=redis_connection)
```

Comment:
Since it's a constant...Also, it would be nice to add the queue name explicitly.

```suggestion
DEFAULT_QUEUE = Queue(name="default", connection=redis_connection)
```